frontend/components/Dashboard/Nav/NavLinks.tsx

```diff
@@ -28,8 +31,8 @@ const NavLinks: React.FC<Props> = ({ onClick }) => {
       <div className="nav-top">
         <Link href="/dashboard/profile">
           <a onClick={onClick}>
-            <img className="profile-img" src="/images/robin-small.png" />
-            <p className="current-user-name">Robin MacPherson</p>
+            <img className="profile-img" src={currentUser.profileImage} />
```

Comment:
Since this is nullable, this should probably be `{currentUser.profileImage && (<img....`

Reply:
Ah, good point. I think I may have another idea I like a wee bit better, though! Just a moment

Reply:
Voilà !! Whatcha think?

![image](https://user-images.githubusercontent.com/34203886/85216938-a0c81480-b33f-11ea-8247-3f4de0dfc869.png)

Reply:
Looks awesome! If you want to make it reusable and take a size prop, we could start adding profile images next to comments too, although that can happen as part of a separate PR if you want.

Reply:
Yeah that would be amazing! Let's do that in a separate PR, I was thinking of making another PR to fix up some of the UI styling that's gone a bit wonky (the color of the dashboard nav and header elements, for example) so perhaps could just include some other improvements in there, too! 

Merging in for nowwww. Grab your things hedgehog!  🏎🧵...🦔🧳💨